src/common

```diff
@@ -229,23 +229,16 @@ OBJ += $(patsubst %.cc, %$(OBJEXT), $(filter %.cc, $(SRC)))
 
 # get version from git
 GIT_INFO = $(shell git describe --tags --always --dirty || echo "n/a")
-GIT_INFO_FILE = .release_info
+RELEASE_INFO = "\#define CBMC_VERSION \"$(CBMC_VERSION) ($(GIT_INFO))\""
+GIT_INFO_FILE = version.h
 
-CBMC_VERSION_FILES += $(filter %_parse_options$(OBJEXT), $(OBJ))
+$(GIT_INFO_FILE):
```

Comment:
Doesn't this target need to be marked `.PHONEY` to ensure it always gets rebuilt? Otherwise I would have thought that after the first make, it wouldn't get rebuilt again because it doesn't depend on anything else.

Reply:
.phoney marks "pseudo targets", i.e., ones that aren't files. This one is an actual file!